examples/kitty/Makefile

```diff
@@ -289,7 +289,17 @@ $(IMAGE_FILE) $(REPORT_FILE): $(addprefix $(BUILD_DIR)/, $(IMAGES)) kitty.system
 
 FORCE: ;
 
+init-submodules:
```

Comment:
Why not keep it consistent with the webserver example?

Reply:
Because in the webserver example you'll have to do `make submodule`, then `make` commands separately. This way you can do just  `make submodule` and it'll build as well. Do you prefer doing it in two distinct steps?

Reply:
If you want to do everything you can do `make submodule all`. I prefer the two distinct steps since it's clear what's doing.

For me at least, it's not that intuitive to have `make submodules` also start compiling code.

Reply:
Alright, I've updated that. I'll quickly update the docs PR as well.